syntax = "proto3";

package myprotobuf;

message Data3d {
    float x = 1;
    float y = 2;
    float z = 3;
}
message Position {
    uint64 sensorId = 1;
    optional uint64 timestamp_ms = 2;
    optional Data3d position_m   = 3;
}